chapters/synchronous.tex

```diff
@@ -807,10 +807,18 @@ \subsection{Sub-clock conversion operators}\doublelabel{sub-clock-conversion-ope
 \\ \hline
 \textbf{noClock}(u)
 &
-The clock of y = \textbf{noClock}(u) is always inferred. At every tick
-of the clock of y, the operator returns the value of u from the last
-tick of the clock of u. If \textbf{noClock}(u) is called before the
-first tick of the clock of u, the start value of u is returned.\\ \hline
+The operator \textbf{noClock} is meant to enable efficient data exchange 
```

Comment:
This should be in the non-normative text.

Reply:
Agree